discord/client.py

```diff
@@ -644,8 +644,25 @@ async def start(self, token: str, *, reconnect: bool = True) -> None:
             An unexpected keyword argument was received.
         """
         await self.login(token)
+        await self.setup()
         await self.connect(reconnect=reconnect)
 
+    async def setup(self) -> Any:
+        """|coro|
+
+        A coroutine to be called to setup the bot, by default this is blank.
+
+        To perform asynchronous setup after the bot is logged in but before
+        it has connected to the Websocket, overwrite this coroutine.
+
+        This is only called once, and will be called before any events are
+        dispatched, making it a better solution than doing such setup in the
+        :func:`on_ready` event.
```

Comment:
```suggestion
        :func:`~discord.on_ready` event.
```

This will fix the build error.